include/game.h

```diff
@@ -721,7 +722,7 @@ typedef struct Entity {
     /* 0x1C */ s16 rotY;
     /* 0x1E */ s16 rotZ;
     /* 0x20 */ s16 rotPivotX;
-    /* 0x22 */ s16 rotPivotY;
+    /* 0x22 */ u16 rotPivotY;
```

Comment:
Why only change the type for `rotPivotY` and not `rotPivotX`?

Reply:
Probably because Y only goes up from zero while X can be positive or negative for left/right.

Either way, this only matches if it's done this way. Most importantly, line 544 of src/dra/90264.c is `self->rotPivotX = -8;` so that one is definitely signed. But rotPivotY has to be unsigned or this function (the one in this PR) explodes.

Reply:
Did we do the psp version too? Does it need to be unsigned here?

Reply:
PSP is here: https://decomp.me/scratch/MOeKz

I never got a good match on it so I don't think we can count on it to say one way or another.

Reply:
Is this the ps1 scratch? https://decomp.me/scratch/TdHwY Unless I'm missing something it seems like s16 works? Changing it to u16 on the psp scratch definitely matches less

Reply:
I guess I got things mixed up, you're both right - this doesn't need to change. Reverted back, now both rotPivot values are s16.